images/macos/macos-10.15-Readme.md

```diff
@@ -92,28 +93,29 @@
 - Sox 14.4.2
 - Subversion (SVN) 1.14.2
 - Switchaudio-osx 1.1.0
-- Vagrant 2.3.0
+- Vagrant 2.3.1
 - VirtualBox 6.1.38r153438
-- yq (https://github.com/mikefarah/yq/) version 4.27.5
+- yq (https://github.com/mikefarah/yq/) version 4.28.1
 - zstd 1.5.2
 
 ### Tools
-- Aliyun CLI 3.0.125
+- Aliyun CLI 3.0.131
 - App Center CLI 2.11.0
-- AWS CLI 2.7.31
-- AWS SAM CLI 1.56.1
-- AWS Session Manager CLI 1.2.339.0
+- AWS CLI 2.8.3
+- AWS SAM CLI 1.60.0
+- AWS Session Manager CLI 1.2.398.0
 - Azure CLI (azure-devops) 0.25.0
-- Azure CLI 2.40.0
-- Bicep CLI 0.10.13
+- Azure CLI 2.41.0
+- Bicep CLI 0.11.1
 - Cabal 3.6.2.0
-- Cmake 3.24.1
-- Fastlane 2.209.1
+- Cmake 3.24.2
+- Colima 
```

Comment:
looks broken to me

Reply:
@bogdan-damian-bgd , we should exclude colima for macos-10.15 - https://github.com/actions/runner-images/blob/5b24a10e54c71e3462e2ab9c0b70ca337e69211f/images/macos/software-report/SoftwareReport.Generator.ps1#L186